src/client-functions/selectors/selector-text-filter.js

```diff
@@ -29,12 +29,12 @@ export default function selectorTextFilter (node, index, originNode, textFilter)
 
         // NOTE: In Firefox, <option> elements don't have `innerText`.
         // So, we fallback to `textContent` in that case (see GH-861).
-        if (node.tagName.toLowerCase() === 'option') {
-            const textContent = node.textContent;
+        // SVG elements do not have `innerText` property as well
 
-            if (!text && textContent)
-                text = textContent;
-        }
+        const elementsWithoutInnerText = ['option', 'svg'];
```

Comment:
Can we simplify this to `const text = node.innerText | node.textContent;`?

Reply:
I think we can, I just didn't want to break the previous approach which is more explicit. Both these approaches will work correctly